server/src/infra/repositories/asset.repository.ts

```diff
@@ -540,11 +543,9 @@ export class AssetRepository implements IAssetRepository {
         .andWhere('person.id = :personId', { personId });
     }
 
-    // Hide stack children only in main timeline
-    // Uncomment after adding support for stacked assets in web client
-    // if (!isArchived && !isFavorite && !personId && !albumId && !isTrashed) {
-    //   builder = builder.andWhere('asset.stackParent IS NULL');
-    // }
+    if (!isArchived && !isFavorite && !personId && !albumId && !isTrashed) {
```

Comment:
This looks crazy. Why don't we just pass an `withStacked` on the main timeline route, probably from the client.